pmlc/dialect/tile/ir/ops.td

```diff
@@ -90,10 +90,14 @@ def GatherOp : MaterializeOp<"gather"> {
   let summary = "special gather operation";
   let arguments = (ins
     RankedTensorOf<[AnyScalar]>:$tensor,
-    RankedTensorOf<[AnyInteger]>:$dims);
+    RankedTensorOf<[AnyScalar]>:$indices,
```

Comment:
I _think_ we have been trying to standardize our spelling to `indexes` instead of `indices`. But perhaps that's different in the MLIR codebase? Does anyone know if upstream MLIR has an opinion on the indexes/indices spelling?

Reply:
I've been attempting to use `indices` everywhere we spell it out, and using `idxs` for shorthand variable names.